Content.Server/Labels/Label/LabelSystem.cs

```diff
@@ -27,13 +27,20 @@ public override void Initialize()
             base.Initialize();
 
             SubscribeLocalEvent<LabelComponent, ExaminedEvent>(OnExamine);
+            SubscribeLocalEvent<LabelComponent, MapInitEvent>(OnLabelCompMapInit);
             SubscribeLocalEvent<PaperLabelComponent, ComponentInit>(OnComponentInit);
             SubscribeLocalEvent<PaperLabelComponent, ComponentRemove>(OnComponentRemove);
             SubscribeLocalEvent<PaperLabelComponent, EntInsertedIntoContainerMessage>(OnContainerModified);
             SubscribeLocalEvent<PaperLabelComponent, EntRemovedFromContainerMessage>(OnContainerModified);
             SubscribeLocalEvent<PaperLabelComponent, ExaminedEvent>(OnExamined);
         }
 
+        private void OnLabelCompMapInit(EntityUid uid, LabelComponent component, MapInitEvent args)
+        {
+            component.CurrentLabel ??= string.Empty;
+            component.CurrentLabel = Loc.GetString(component.CurrentLabel);
```

Comment:
Perhaps this should be a null check instead of a set to empty string? This code kind of confuses the meaning of a `null` label vs an empty string label (and in most place they're considered to mean the same thing).

Reply:
I mean, I can change it but I don't think it really makes a difference - I don't expect that string to ever be null. You would have had to intentinally map a Label with no text assigned. It's just there so the code can theoretically handle that situation. 

If there is a more proper format I can change it (and the beacon one too) so it does not end up being copied further

Reply:
It matters because `OnExamine` checks for null (and not for empty string). So please change this.

Reply:
No longer modifying the string since at that point there is no point in running localization anyway. So there is no different behavior now (the labeler can't have a null string and an empty one removes the label)
I have added comments on the component